README.md

```diff
@@ -16,35 +16,44 @@ To generate multi-view images from a single input image, you can run the followi
 import torch
 import requests
 from PIL import Image
-from diffusers import DiffusionPipeline, EulerAncestralDiscreteScheduler
+from diffusers import DiffusionPipeline, EulerAncestralDiscreteScheduler, ControlNetModel
+
+# Define constants or variables with meaningful names
+NUM_INFERENCE_STEPS = 36
+CONDITIONING_SCALE = 0.75
+MODEL_URL = "https://d.skis.ltd/nrp/sample-data/0_cond.png"
```

Comment:
This should be `COND_URL`, `INPUT_URL` or something like that. Moreover, the new structure makes changing to a local file input harder.